lisa/runner.py

```diff
@@ -122,13 +113,20 @@ def _initialize_runners(self) -> None:
         # group filters by runner type
         runner_filters: Dict[str, List[schema.BaseTestCaseFilter]] = {}
         for raw_filter in self._runbook.testcase_raw:
-            runner_type = raw_filter.get(constants.TYPE, constants.TESTCASE_TYPE_LISA)
-            raw_filters: List[schema.BaseTestCaseFilter] = runner_filters.get(
-                runner_type, []
-            )
-            if not raw_filters:
-                runner_filters[runner_type] = raw_filters
-            raw_filters.append(raw_filter)
+            # by default run all specified case unless 'enable' is specified as false
+            filter = schema.BaseTestCaseFilter.schema().load(raw_filter)  # type:ignore
+            if filter.enable:
+                runner_type = raw_filter.get(
```

Comment:
since the filter is converted, so `runner_type` is not useful. When it's needed, use `filter.type` is enough.